xline/src/storage/kvwatcher.rs

```diff
@@ -564,20 +576,16 @@ mod test {
         let handle = tokio::spawn({
             let store = Arc::clone(&store);
             async move {
-                let mut revision = 2;
+                // let mut revision = 2;
```

Comment:
remove this line of code if we don't need it.

Reply:
Removed